js/main.js

```diff
@@ -1,2 +1,32 @@
 /* global data */
 /* exported data */
+var $inputUrl = document.querySelector('input');
+var $image = document.querySelector('img');
+var $form = document.querySelector('form');
+
+function addImage(event) {
+
+  $image.setAttribute('src', $form.elements.image.value);
+
+}
+
+$inputUrl.addEventListener('input', addImage);
+
+function inputEntry(event) {
+  event.preventDefault();
+  var entry = {
+    image: $form.elements.image.value,
+    title: $form.elements.title.value,
+    notes: $form.elements.notes.value
+  };
+
+  entry.entryId = data.entries.length + 1;
+  entry.nextEntryId = data.entries.length + 2;
```

Comment:
This should just be incrementing instead of using the `length` of the `data.entries`. Also, the `entry` should not receive a `nextEntryId`.

```js
data.nextEntryId++;
```

Reply:
![Screen Shot 2021-02-10 at 12 15 57 PM](https://user-images.githubusercontent.com/74885918/107566454-c9846f00-6b99-11eb-9d02-d19140e417e7.png)